recipes/pychrono/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pychrono" %}
+{% set version = "4.0.0" %}
+{% set build = 0 %}
+{% set sha256 = "803b5089e48334771f652f5bec79f95f2a42cfb02d2ffbaf8ad9e5dca2ba1886" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/projectchrono/chrono/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - swig 3.0.12
+    - pcre
+    - bzip2
+    - python
+    - zlib
```

Comment:
4 lines above should be in host.

Reply:
sorry I missed that.

Reply:
No need to be sorry. Thanks for reviewing.

Reply:
I will fixup the feedstock as soon as it is created.